### ISSUE TITLE
Invalid HTML. Replace <span> with <div> for <v-date-picker>

### ISSUE THREAD

**Issue by geoffreyrose** [nathanreyes/v-calendar#831]
Consider using a `<div>` instead of a `<span>` for `<v-date-picker>`  since it is technically invalid html to have a `block` element inside of a `inline` element. When someone adds a `div` or another `block` element inside of `<v-date-picker>` it is creating invalid html. While most browser deal with this without issue, it would be better to have valid html.

**Comment by mickaelponsolle** [comment]
Indeed, I have a warning in the console when I refreshed a page containing `<v-date-picker>`. It should be nice if a `<div>` will be used instead of a `<span>`. 
Warning details : The client-side rendered virtual DOM tree is not matching server-rendered content. This is likely caused by incorrect HTML markup, for example nesting block-level elements inside `<p>`, or missing `<tbody>`. Bailing hydration and performing full client-side render.

**Comment by georgwittberger-t-systems-mms** [comment]
We are having the same problem in our project at the moment. @nathanreyes, are there any plans to release a new version with your committed change soon?

**Comment by uglu** [comment]
Please fix this as soon as possible. It's an HTML violation to have flow content inside of phrasing content, e.g `<div>` inside of `<span>`

[Example in datepicker](https://github.com/nathanreyes/v-calendar/blob/a4e9bceb168aa302eeff5143d41d057c1c5ee547/src/components/DatePicker.vue#L126)

**Comment by Pete0000** [comment]
Please also add a class to the wrapper element so we style it easier
